docs/cross-contract/callbacks.md

```diff
@@ -46,61 +46,125 @@ mod ext_calculator {
 Let's assume the calculator is deployed on `calc.near`, we can use the following:
 
 ```rust
-const CALCULATOR_ACCOUNT_ID: &str = "calc.near";
-const NO_DEPOSIT: Balance = 0;
-const BASE_GAS: Gas = 5_000_000_000_000;
+const BASE_GAS: Gas = Gas(5_000_000_000_000);
 
 #[near_bindgen]
 impl Contract {
     pub fn sum_a_b(&mut self, a: U128, b: U128) -> Promise {
-        let calculator_account_id: AccountId = CALCULATOR_ACCOUNT_ID.to_string();
-        ext_calculator::sum(a, b, &calculator_account_id, NO_DEPOSIT, BASE_GAS)
+        let calculator_account_id: AccountId = "calc.near".parse().unwrap();
+        // Call the method `sum` on the calculator contract. Attach static GAS equal to BASE_GAS.
+        // Any unused GAS will be added as well since the default GAS weight is 1.
+        // Attached deposit is defaulted to 0.
+        ext_calculator::ext(calculator_account_id)
+            .with_static_gas(BASE_GAS)
```

Comment:
optional: we can omit the static gas for this, doesn't seem necessary in favor of more concise code

Reply:
Fixed